tiledb/common/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if (TILEDB_TESTS)
 endif()
 
 get_gathered(COMMON_SOURCES)
+list(REMOVE_ITEM COMMON_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/governor/governor.cc)
```

Comment:
This line is a symptom that's something's wrong. It can't stay. If an integration with `baseline` is requiring _ad hoc_ exceptions like this, we need to get whatever's causing the problem changed and not just mask over the problem with a statement like this one.